way_to_home/user/models.py

```diff
@@ -1 +1,55 @@
 """This module implements class that represents the user entity."""
+
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.db import models, IntegrityError
+
+
+class User(AbstractBaseUser):
+    """Model for User entity."""
+
+    email = models.EmailField(max_length=64, unique=True)
+    password = models.CharField(max_length=128)
+    google_token = models.CharField(blank=True, max_length=255)
```

Comment:
I guess you should replace blank to null = True 
Add unique=True

Reply:
Done